docs/en/modules/enduser-setup.md

```diff
@@ -1,13 +1,14 @@
 # enduser setup Module
 | Since  | Origin / Contributor  | Maintainer  | Source  |
 | :----- | :-------------------- | :---------- | :------ |
-| 2015-09-02 | [Robert Foss](https://github.com/robertfoss) | [Robert Foss](https://github.com/robertfoss) | [enduser_setup.c](../../../app/modules/enduser_setup.c)|
+| 2015-09-02 | [Robert Foss](https://github.com/robertfoss), [Jason Follas](https://github.com/jfollas) | [Robert Foss](https://github.com/robertfoss) | [enduser_setup.c](../../../app/modules/enduser_setup.c)|
```

Comment:
The use of the terms "Origin / Contributor" may not be very obvious here, I agree. It isn't documented anywhere either. However, we've been using it like this:
- origin: where the code originally branched off from. Many of our modules where ported to NodeMCU from somewhere, often from Arduino.
- contributor: the developer(s) who contributed the original NodeMCU code (i.e. often ported code).

→ "Origin / Contributor" are the same for NodeMCU modules like EUS which started from scratch
→ "Origin / Contributor" usually don't change - even though you greatly contributed to EUS, thanks again 🍺 

Maybe we should change this pattern.

Even better would be if you allowed us to list you as the maintainer 😃 It seems @robertfoss didn't have enough spare time recently to maintain it.